paper/paper.md

```diff
@@ -57,33 +57,33 @@ bibliography: paper.bib
 
 # Summary
 
-Mass spectrometry data is at the heart of numerable applications in the biomedical and life sciences.
-With growing use of high throughput techniques researchers need to analyse larger and more complex datasets. In particular through joint effort in the research community, fragmentation mass spectrometry datasets are growing in size and number.
+Mass spectrometry data is at the heart of numerous applications in the biomedical and life sciences.
+With growing use of high-throughput techniques, researchers need to analyze larger and more complex datasets. In particular through joint effort in the research community, fragmentation mass spectrometry datasets are growing in size and number.
 Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLights [@haug_metabolights_2020] serve as an open-access hub for sharing of raw, processed, or annotated fragmentation mass spectrometry data (MS/MS).
```

Comment:
"mass spectrometry" and "(MS/MS)" are not equivalent terms but the current formulation does suggest that.

Reply:
Yes, MS/MS stands for tandem mass spectrometry I believe. But "fragmentation mass spectrometry" and MS/MS are also often used as equivalents although that might technically not be correct... but I am not the expert here (I'll check).

Reply:
I removed the (MS/MS) here since it is not necessary anyway.